src/test/kotlin/io/spine/protobuf/DescriptorExtsSpec.kt

```diff
@@ -55,4 +58,20 @@ internal class DescriptorExtsSpec {
             descriptor.field(SECONDS_FIELD_NUMBER)!!.number shouldBe SECONDS_FIELD_NUMBER
         }
     }
+
+    @Test
+    fun `list nested types excluding synthetic ones for map entries`() {
+        val descr = AddressBook.getDescriptor()
```

Comment:
What if `AddressBook` has a nested `RecentEntry` message, occupying the name of the map entry?

Reply:
This is prohibited by [Protobuf specs](https://protobuf.dev/programming-guides/proto3/#maps-features).